docs/application/data_pipeline_template.md

```diff
@@ -0,0 +1,162 @@
+- Graph: creat Rayleigh plots
+- Reports: prepare TAUW report
+- APIs that serve the processed data to other services or researchers
+
+
+## End-to-End Example of the Data Pipeline
+1. **Ingestion:**
+```python
+from mibipret.data.load_data import load_csv
+from mibipret.data.load_data import load_excel
+
+# load data from csv file
+file_path = 'path/to/data/data.csv'
+data_raw,units = load_csv(file_path,verbose=False)
+
+# load data from excel file per sheet 
+file_path = 'path/to/data/data.xlsx'
+environment_raw,units = load_excel(file_path, sheet_name = 'environment', verbose = verbose)
+```
+    
+2. **Preprocessing:(Cleaning and Standardization)**
+
+```Python
+from mibipret.data.check_data import standardize
+
+# Runs all checks on data, i.e. column names (check_columns()), data format (check_data_frame()), units (check_units()), names (standard_names()) and values (check_values()) in one go and returns transformed data with standard column names and valueas in numerical type where possible. Data is reduced to those columns containing known quantities if reduce=true.
+data, units = standardize(data_raw, reduce = True, verbose=False)
+```
+
+3. **Transformation:(processing and enrichment)**
+```python
+ # For NA screening, stochiometric equations are used to analyze electron balance, here is how to perform NA screening step by step:
+ import mibipret.analysis.sample.screening_NA as na
+ #Calculation of number of electrons for reduction
+ #Returns pandas-Series with total amount of electron reductors per well in [mmol e-/l]:
+    tot_reduct = na.reductors(data,verbose = True,ea_group = 'ONSFe')                               
+
+
+#Calculation of number of electrons needed for oxidation
+#Returns pandas-Series with total amount of oxidators per well in [mmol e-/l]:
+    tot_oxi = na.oxidators(data,verbose = True, contaminant_group='BTEXIIN')                              
+
+
+#Calculation of number of electron balance
+#Returns pandas-Series with ratio of reductors to oxidators. If value below 1, available electrons for reduction are not sufficient for reaction and thus NA is potentially not taking place:
+    e_bal = na.electron_balance(data,verbose = True)                               
+
+# Evaluation of intervention threshold exceedance
+#Calculation of total concentration of contaminants/specified group of contaminants
+#Returns pandas-Series with total concentrations of contaminants per well in [ug/l]:
+    tot_cont = na.total_contaminant_concentration(data,verbose = True,contaminant_group='BTEXIIN')                             
+ 
+ # If you want to perform complete NA screening and evaluation of intervention threshold exceedance in one go:
+ data_na = na.screening_NA(data,verbose = True)
+
+ # It is also possible to run full NA screening with results added to data using argument (inplace = True):
+ na.screening_NA(data,inplace = True,verbose = False)
+```
+4. **Storage:**
+`Not available`

```

Comment:
Markdown has some nice syntax to create warning boxes that stand out and help warn the reader. 

```suggestion
!!! Warning
    Mibipret does not have support for file storage
```